client/src/Backoffice/components/Main/components/modals/AddCustomer/MedicalRequirements/MedicalRequirements.tsx

```diff
@@ -40,97 +38,95 @@ function MedicalRequirements() {
-              control={<Checkbox />}
-            />
-          </div>
+    <div className="">
+      <FormControlLabel
+        control={
+          <Checkbox
+            checked={isServiceForSelf}
+            onChange={(e) => setIsServiceForSelf(e.target.checked)}
+            name="shipment"
+          />
+        }
+        label="השירות מיועד לטיפול עבור עצמי"
+      />
+      <div className="flex gap-4 mt-6">
+        <div className="flex flex-col gap-4 flex-1">
+          <TextField
+            required
+            id="outlined-basic"
+            label="המטופל שם פרטי"
+            variant="outlined"
+            fullWidth
+            disabled={isServiceForSelf}
+          />
+          <TextField
+            required
+            id="outlined-basic"
+            label="המטופל תעודת זהות"
+            variant="outlined"
+            fullWidth
+            disabled={isServiceForSelf}
+          />
+          <TextField id="outlined-basic" label="בניין בביה״ח" variant="outlined" fullWidth />
+          <DatePicker
+            value={value}
+            onChange={(newValue) => setValue(newValue)}
+            label="תחילת התקופה בה תזדקקו לשירות ההסעות"
+            disablePast
+            format="DD/MM/YYYY"
+            slotProps={{
+              textField: {
+                required: true
+              }
+            }}
+          />
+          <TextField
+            id="outlined-basic"
+            required
+            label="הסיבה לשימוש בשירות ההסעות"
+            variant="outlined"
+            fullWidth
+            multiline
+            maxRows={4}
+          />
+        </div>
+        <div className="flex flex-col gap-4 flex-1">
+          <TextField
+            required
+            id="outlined-basic"
+            label="המטופל שם משפחה"
+            variant="outlined"
+            fullWidth
+            disabled={isServiceForSelf}
+          />
+          <TextField id="outlined-select-currency" select label="בית חולים לאיסוף והורדה">
+            {hospitals.map((option) => (
+              <MenuItem key={option.value} value={option.value}>
+                {option.label}
+              </MenuItem>
+            ))}
+          </TextField>
+          <TextField id="outlined-basic" required label="מחלקה" variant="outlined" fullWidth />
+          <DatePicker
+            value={value}
+            onChange={(newValue) => setValue(newValue)}
+            label="סיום התקופה בה תזדקקו לשירות ההסעות"
+            views={['year', 'month', 'day']}
+            disablePast
+            format="DD/MM/YYYY"
+            slotProps={{
+              textField: {
+                required: true
+              }
+            }}
+          />
+          <FormControlLabel
+            label="הנני מאשר/ת כי קראתי את תקנון האתר ואת מדיניות הפרטיות ומסכים לתנאיהם"
+            control={<Checkbox />}
+          />
         </div>
       </div>
-    </LocalizationProvider>
+    </div>
```

Comment:
@aviv-taori Have you tested this? It seems like you have forgotten to remove the extra `div` here.

Reply:
No unnecessary `div,` anyway not my component I just deleted `</LocalizationProvider>`